capstone/src/test/java/com/google/sps/VoterServletTest.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps;
+
+import com.google.sps.servlets.VoterServlet;
+import java.io.IOException;
+import java.net.*;
+import java.util.*;
+import org.json.*;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class VoterServletTest {
+  @Test
+  public void rightOutput() throws IOException {
+    VoterServlet voting = new VoterServlet();
+    String API_key = "";
+    String testElectionId = "2000";
+    String testAddress = "1000 17th Ave N";
+    String charset = "UTF-8";
+    String testQuery =
+        String.format(
+            "key=%s&address=%s&electionId=%s",
+            URLEncoder.encode(API_key, charset),
+            URLEncoder.encode(testAddress, charset),
+            URLEncoder.encode(testElectionId, charset));
+    URLConnection testconnection =
+        new URL("https://www.googleapis.com/civicinfo/v2/voterinfo" + "?" + testQuery)
+            .openConnection();
+    JSONObject testelectionJsonObject = new JSONObject();
```

Comment:
With the refactors to the other file, you should be able to get rid of lines 18-30